database/migrations/2022_04_06_152416_add_uuids_to_tables.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public const MODEL_CLASSES = [
+        \App\Models\Article::class,
+        \App\Models\Thread::class,
+        \App\Models\Reply::class,
+    ];
```

Comment:
I don't think referencing app code is wise in migrations. Let's reference the tables directly.